.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.uc binary
```

Comment:
This is probably unnecessary, but it doesn't hurt to tell git about our custom binary filetype since it's not very common.